third_party/rust/adblock/v0_8/crate/src/cosmetic_filter_cache.rs

```diff
@@ -248,6 +248,8 @@ impl CosmeticFilterCache {
         let mut script_injections = HashMap::<&str, PermissionMask>::new();
         let mut exceptions = HashSet::new();
 
+        let mut except_all_scripts = false;
```

Comment:
I know this is part of the change adblock-rs, but this variable name is very confusing imo and should either be changed or a comment added in a follow-up for that repo